### ISSUE TITLE
CyanideAndHappiness, Dilbert and IAmArg are not updating.

### ISSUE THREAD

**Issue by kierun** [wummel/dosage#102]
I cannot seem to be getting any new CyanideAndHappiness, Dilbert or IAmArg comics.

    Dilbert> ERROR: Patterns ['<\\s*[aA]\\s+(?:[^>]*\\s+)?[hH][rR][eE][fF]\\s*=\\s*"(/\\d+-\\d+-\\d+/)"[^>]*STR_Prev[^>]*>'] not found at URL http://dilbert.com/.
    CyanideAndHappiness> ERROR: Patterns ['<\\s*[aA]\\s+[^>]*prev[^>]*\\s+[hH][rR][eE][fF]\\s*=\\s*"(/comics/\\d+/)"[^>]*[^>]*>'] not found at URL http://www.explosm.net/comics/.
    IAmArg> ERROR: Patterns ['<\\s*[iI][mM][gG]\\s+(?:[^>]*\\s+)?[sS][rR][cC]\\s*=\\s*"(http\\:\\/\\/iamarg\\.com\\/comics/\\d+-\\d+-\\d+[^"]+)"[^>]*[^>]*>'] not found at URL http://iamarg.com/.

Do I need to get a new definition file?  If so, where from?

Thanks

**Comment by Manabi** [comment]
The current code has CyanideAndHappiness and Dilbert fixed, although IAmArg is still broken. If you're on Linux and installed using pip, you can try `pip install Django --upgrade` to upgrade. I'm not entirely sure it'll work, since I just update my files manually.

You can also just download new copies of c.py and d.py from here: https://github.com/wummel/dosage/tree/master/dosagelib/plugins  That's the current versions of both files.  Just replace the ones you have currently.  On Linux they're usually in /usr/local/lib/python2.7/dist-packages/dosagelib/plugins/ 

You can probably update the files on Windows to if that's what you're using.  But I have no clue where it gets installed.

**Comment by kierun** [comment]
Yeah, updated to the latest code in the git repository: CyanideAndHappiness and Dilbert now work.

Thank you!

**Comment by Manabi** [comment]
IAmArg has been fixed in the new repository, in d89d4e4.

**Comment by kierun** [comment]
Quoth Manabi on Wed, Apr 15, 2015 at 12:10:32 -0700
> IAmArg has been fixed in the new repository, in d89d4e4.

I'll have to wait till next week before I can update my home set up. The birth
of my twin daughters has kinda knocked me off the 'net ;)

If anyone else confirms this as fixed, I am happy for you to close the bug
report.

Thank you for your work! Much appreciated.

-- 
GPG fingerprint = 4ED9 EC45 AEFD A60E 7CE5  A3EC 519C 0845 AFDF 2C31
